features/auth/signup/Activation.tsx

```diff
@@ -0,0 +1,71 @@
+import { CircularProgress, Container } from "@material-ui/core";
+import * as Sentry from "@sentry/react";
+import Alert from "components/Alert";
+import { useAppRouteStyles } from "components/AppRoute";
+import HtmlMeta from "components/HtmlMeta";
+import StyledLink from "components/StyledLink";
+import { useTranslation } from "i18n";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+import { useMutation } from "react-query";
+import { loginRoute } from "routes";
+import { service } from "service";
+import { ActivateUserReq, ActivateUserRes } from "service/auth";
+import { HttpError } from "service/http";
+import isHttpError from "utils/isHttpError";
+import stringOrFirstString from "utils/stringOrFirstString";
+
+export default function Activation() {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+  const classes = useAppRouteStyles();
+
+  const router = useRouter();
+  const activationUID = stringOrFirstString(router.query.uid);
+  const activationToken = stringOrFirstString(router.query.token);
+
+  const {
+    error,
+    isLoading,
+    isSuccess,
+    mutate: activateUser,
+  } = useMutation<ActivateUserRes, HttpError, ActivateUserReq>(
+    ({ uid, token }) => service.auth.activateUser(uid, token)
+  );
+
+  useEffect(() => {
+    if (activationUID && activationToken) {
+      activateUser({ uid: activationUID, token: activationToken });
+    }
+  }, [activateUser, activationUID, activationToken]);
+
+  useEffect(() => {
+    Sentry.captureException(error, {
+      tags: {
+        component: "auth/signup/Activation",
+      },
+    });
+  }, [error]);
```

Comment:
This seems quite an indirect way to log an exception/error with `useEffect`. I'd recommend calling this in the `onError` callback, passed as a config in the second param of the `useMutation` instead (syntax shown [in the docs](https://react-query-v3.tanstack.com/guides/mutations#mutation-side-effects))